### ISSUE TITLE
wit.text_query should return dict instead of str

### ISSUE THREAD

**Issue by kienpham2000** [wit-ai/pywit#9]
res = wit.text_query('turn on the lights in the kitchen', access_token)
print type(res)  

should be dict type instead of str

**Comment by edu2004eu** [comment]
+1

**Comment by walchko** [comment]
it returns a json string ... just use the json library to turn it into a dict:

```python
result = wit.text_query('hello', wit_token )
ans = json.loads(result) 
```

Alot of REST libraries do this, so it is acceptable to return a json string. Plus, there are multiple json libraries so this empowers the developer to pick the library they want to use without forcing them to use a specific one.

**Comment by kienpham2000** [comment]
So if it returns dict by default, we don't have to do extra json parsing :).

**Comment by blandinw** [comment]
Fixed with 2.0
